projects/packages/videopress/src/client/admin/components/input/index.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { SearchIcon } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { useCallback, ChangeEvent, KeyboardEvent } from 'react';
+/**
+ * Internal dependencies
+ */
+import styles from './style.module.scss';
+import { InputProps, SearchInputProps } from './types';
+import type React from 'react';
+
+/**
+ * Input component
+ *
+ * @param {InputProps} props - Component props.
+ * @returns {React.ReactNode} - Input react component.
+ */
+export const Input: React.FC< InputProps > = ( {
+	className,
+	value,
+	placeholder,
+	icon,
+	onChange,
+	onEnter,
+} ) => {
+	const hasIcon = icon != null;
+
+	const handleChangeEvent = useCallback(
+		( e: ChangeEvent< HTMLInputElement > ) => {
+			if ( onChange != null ) {
+				onChange( e.currentTarget.value );
+			}
+		},
+		[ onChange ]
+	);
+
+	const handleKeyDownEvent = useCallback(
+		( e: KeyboardEvent< HTMLInputElement > ) => {
+			if ( onEnter != null && e.key === 'Enter' ) {
+				onEnter( e.currentTarget.value );
+			}
+		},
+		[ onEnter ]
+	);
+
+	return (
+		<div className={ classnames( className, styles.wrapper ) }>
+			{ hasIcon && icon }
```

Comment:
```suggestion
                        { icon }
```

If `icon` is `undefined` or `null`, if you render it, that's no problem, React just ignore it.